### ISSUE TITLE
Bug: ENOENT isn't caught

### ISSUE THREAD

**Issue by OmgImAlexis** [alexcanessa/typescript-coverage-report#17]
```shell
Error: ENOENT: no such file or directory, rmdir '/Users/xo/code/_/coverage-ts'
```

**Comment by alexcanessa** [comment]
@all-contributors please add @OmgImAlexis for bug
